frontend/src/scenes/surveys/SurveyAppearance.tsx

```diff
@@ -145,8 +145,39 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
             )}
             <div className="mt-2">Button text</div>
             <LemonInput
-                value={appearance?.submitButtonText || defaultSurveyAppearance.submitButtonText}
```

Comment:
I had to make this change because, you couldn't remove completely the Submit Button text if you want to update it with a new text.